.github/workflows/main.yml

```
@@ -79,4 +86,33 @@ jobs:
       with:
         push: true
         context: ./src
-        tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.DockerImageName }}:${{ github.sha }}
\ No newline at end of file
+        tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.DockerImageName }}:${{ github.sha }}
+
+  helm-ci:
+    needs: python-ci
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_name == 'main' }}
```

Thus, this will work. For pull requests from a fork, the branch from which the PR is created is usually not the main branch but a branch of the fork, so this condition is not met
```suggestion
    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' || github.event_name == 'pull_request' && github.base_ref == 'main' }})
```